pkg/connservices/proto_converter.go

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connservices
+
+import (
+	"fmt"
+
+	schemav1 "github.com/conduitio/conduit-connector-protocol/proto/schema/v1"
+	"github.com/conduitio/conduit/pkg/foundation/cerrors"
+	"github.com/conduitio/conduit/pkg/schema"
+)
+
+type protoConverter struct {
+}
```

Comment:
I assume you took the `protoConverter` idea from the processors package. I propose to rather move these converters into the connector-protocol and use the same [`fromproto`](https://github.com/ConduitIO/conduit-connector-protocol/tree/main/cplugin/v2/fromproto) and [`toproto`](https://github.com/ConduitIO/conduit-connector-protocol/tree/main/cplugin/v2/toproto) pattern. A nice side-effect of this is that we won't be able to use any types from Conduit, we'll have to use types that are defined in the protocol or in conduit-commons, which will in turn make it possible to reuse them in connectors and processors.